### PR TITLE
Fixes 233 - broken storagePod VM creation

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: lib/fog/vsphere/requests/compute/create_vm.rb [hunk 1]
```diff
@@ -48,14 +48,16 @@ def create_vm_on_datastore(vm_cfg, vmFolder, resource_pool, host = nil)
...
           storage_spec = RbVmomi::VIM::StoragePlacementSpec.new(
             type: 'create',
             folder: vmFolder,
             resourcePool: resource_pool,
-            podSelectionSpec: pod_selection_spec(vm_pod_name, group_disks_by_storage_pod(volumes, vm_pod_name), datacenter),
+            podSelectionSpec: pod_selection_spec(vm_pod_name, disks_per_pod, datacenter),
             configSpec: vm_cfg,
             host: host
           )
+          pp storage_spec
```
Comment (timogoebel) [inline lib/fog/vsphere/requests/compute/create_vm.rb]: Debug code.

File: lib/fog/vsphere/requests/compute/create_vm.rb [hunk 2]
```diff
@@ -86,6 +88,7 @@ def pod_selection_spec(vm_pod_name, disks_per_pod, datacenter, with_relocation:
           raw_pods = {}
           raw_pods[vm_pod_name] = get_raw_storage_pod(vm_pod_name, datacenter)
           disks_per_pod.each_key { |pod_name| raw_pods[pod_name] ||= get_raw_storage_pod(pod_name, datacenter) }
+          pp vm_pod_name, disks_per_pod, raw_pods
```
Comment (timogoebel) [inline lib/fog/vsphere/requests/compute/create_vm.rb]: Debug code.

**Comment by ezr-ondrej** [comment]
Thanks @timogoebel, removed the debug.

**Comment by ezr-ondrej** [comment]
Yeah, ok to test

**Pull request merged**